pkg/agent/openflow/client.go

```diff
@@ -236,3 +252,56 @@ func (c *client) Initialize(roundNum uint64) error {
 	}
 	return nil
 }
+
+func (c *client) Initialize(roundNum uint64) (<-chan struct{}, error) {
+	// Initiate connections to target OFswitch, and create tables on the switch.
+	connCh := make(chan struct{})
+	if err := c.bridge.Connect(maxRetryForOFSwitch, connCh); err != nil {
+		return nil, err
+	}
+
+	// Ignore first notification, it is not a "reconnection".
+	<-connCh
+
+	c.cookieAllocator = cookie.NewAllocator(roundNum)
+
+	return connCh, c.initialize()
+}
+
+func (c *client) Reconcile() {
```

Comment:
This is different from reconciliation after agent restarts. Would you use some way to differentiate these two?